Material.Styles/Assists/ButtonAssist.cs

```diff
@@ -31,5 +33,48 @@ public static void SetClickFeedbackColor(AvaloniaObject element, IBrush value) =
             element.GetValue<IBrush?>(ClickFeedbackColorProperty);
 
         #endregion
+        
+        // Used for memorise which hyperlink button clicked (or just say, visited)
+        #region AttachedProperty : IsClicked
+
+        public static readonly AvaloniaProperty<bool?> IsClickedProperty =
```

Comment:
What's the point of this attached property, if we have an IsVisited inside a HyperlinkButton itself?